docs/releases.md

```diff
@@ -0,0 +1,35 @@
+# Releases
+
+[Latest releases](https://github.com/argoproj/argo-events/releases)
+
+## Supported Versions
+
+Versions are expressed as x.y.z, where x is the major version, y is the minor
+version, and z is the patch version, following Semantic Versioning terminology.
+
+We maintain release branches for the most recent two minor releases.
+
+Fixes may be backported to release branches, depending on severity, risk, and,
+feasibility.
+
+If a release contains breaking changes, or CVE fixes, this will documented in
+the release notes.
+
+## Supported Version Skew
+
+Image versions of `eventsource`, `sensor`, `eventbus-controller`,
+`eventsource-controller`, `sensor-controller` and `events-webhook` should be the
+same.
+
+## Release Cycle
+
+For **unstable**, we build and tag `latest` images for every commit to master.
+
+New major versions are released roughly every 6 months. Release candidates for
```

Comment:
@alexec - Do you think this is good?

Reply:
* v1.2.0 = Jan 2021
* v1.1.0 = Nov 2020
* v1.0.0 = Sep 2020

That's about every 3 months?

Reply:
hmmm. Major means `v1.x.x`, `v2.x.x`, I can not tell how frequently it would be for major releases. Let's not make that statement but only do minor release (every 2 months), is that ok?